core/translate/expr.rs

```diff
@@ -698,7 +698,93 @@ pub fn translate_expr(
             }
             Ok(target_register)
         }
-        ast::Expr::Case { .. } => todo!(),
+        ast::Expr::Case {
+            base,
+            when_then_pairs,
+            else_expr,
+        } => {
+            // There's two forms of CASE, one which does quality against the when values:
```

Comment:
Typo: equality

I would also phrase it "one which checks a base expression for equality against the WHEN values:"

and then add an example:

`example: CASE employee.position WHEN 'manager' THEN '$$$' WHEN 'IC' then '$$' ELSE '$' END`

Reply:
Appreciate the comment otherwise though, comments are great!